bibliopixel/drivers/network.py

```diff
@@ -58,6 +68,15 @@ def _send_packet(self):
             raise IOError(error)
 
     def set_device_brightness(self, brightness):
+        """Hardware specific method to set the global brightness for
+        this driver's output. This method is required to be implemented,
```

Comment:
"This method is required to be implemented" - the reader doesn't care.

Rather say:  "FOR INTERNAL USE ONLY." (or whatever phrase) Users should call :py:meth:`.driver_base.DriverBase.set_brightness` instead."